### PR TITLE
Fixed all pep8 errors

### PR DESCRIPTION
**Pull request by rohithasrk**
This PR aims to fix all the pep8 errors in the package. I basically used the autopep8 tool in order to do so. The code now seems consistent with the style guidelines and is a little beautified.

### CODE REVIEW

File: django_netjsongraph/admin.py [hunk 1]
```diff
@@ -26,8 +32,13 @@ def _message(self, request, rows, suffix, level=messages.SUCCESS):
         if rows == 1:
             prefix = _('1 {0} was'.format(self.model._meta.verbose_name))
         else:  # pragma: nocover
-            prefix = _('{0} {1} were'.format(rows, self.model._meta.verbose_name_plural))
-        self.message_user(request, '{0} {1}'.format(prefix, suffix), level=level)
+            prefix = _(
+                '{0} {1} were'.format(
+                    rows,
+                    self.model._meta.verbose_name_plural))
```
Comment (nemesisdesign) [inline django_netjsongraph/admin.py]: this kind of line breaking is unreadable

File: django_netjsongraph/admin.py [hunk 2]
```diff
@@ -26,8 +32,13 @@ def _message(self, request, rows, suffix, level=messages.SUCCESS):
...
+        self.message_user(
+            request, '{0} {1}'.format(
```
Comment (nemesisdesign) [inline django_netjsongraph/admin.py]: same

**Comment by rohithasrk** [comment]
I see. Will look into these once the GSoC applications are done. @nemesisdesign

**Comment by nemesisdesign** [comment]
@rohithasrk ok, no hurry.

**Comment by rohithasrk** [comment]
@nemesisdesign I used flake8 and it did not report many errors. Used isort on top of it. If flake8 and isort are enough for styling, I think this is it.

**Comment by rohithasrk** [comment]
@nemesisdesign There is a tiny import error that flake8 shows up. Please visit https://travis-ci.org/netjson/django-netjsongraph/jobs/229379983. Want to know if this should be ignored.

File: tests/local_settings.example.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # RENAME THIS FILE TO local_settings.py IF YOU NEED TO CUSTOMIZE SOME SETTINGS
 # BUT DO NOT COMMIT
 
-#DATABASES = {
+# DATABASES = {
```
Comment (nemesisdesign) [inline tests/local_settings.example.py]: cancel changes to this file, and include this file to the excluded files of flake8 and isort

File: tests/settings.py [hunk 4]
```diff
@@ -81,7 +81,8 @@
             'format': '[%(levelname)s] %(message)s'
         },
         'verbose': {
-            'format': '\n\n[%(levelname)s %(asctime)s] module: %(module)s, process: %(process)d, thread: %(thread)d\n%(message)s'
```
Comment (nemesisdesign) [inline tests/settings.py]: same as previous, cancel changes to this file, and include this file to the excluded files of flake8 and isort

File: runflake8 [hunk 5]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -e
+
+flake8 --max-line-length=110 \
+       --exclude=./docs/,./build/,./setup.py,./django_netjsongraph/migrations/ || exit 1
```
Comment (nemesisdesign) [inline runflake8]: See https://github.com/openwisp/django-netjsonconfig/blob/master/runflake8#L4 for an example of how to exclude test settings files

File: tests/local_settings.example.py (hunk 3 above)
Comment (rohithasrk) [inline tests/local_settings.example.py]: Okay.

File: tests/settings.py (hunk 4 above)
Comment (rohithasrk) [inline tests/settings.py]: I see. Will do it

File: runflake8 (hunk 5 above)
Comment (rohithasrk) [inline runflake8]: Yeah. I understand. @nemesisdesign: Should I exclude the build files too? I get the lint errors in my locally set up application but they wouldn't pop up in Travis as build files are git ignored.

**Comment by nemesisdesign** [comment]
@rohithasrk create a virtualenv that uses python3 and sort imports using that, otherwise the build will complain. Also squash all the commits.

**Comment by rohithasrk** [comment]
@nemesisdesign: Yes. I get it. And I'll squash all of them once the build is error free.

**Pull request merged**